src/views/Summary.js

```diff
@@ -12,32 +10,26 @@ const Summary = () => {
   const [totSavings, setTotSavings] = useState(dummyTotSavings);
 
   return (
-    <div className="container-md">
-      <header className="row text-center my-5">
-        <h1 className="display-1 text-primary">Mat's Budget</h1>
-      </header>
-      <section className="row my-4">
-        <div className="text-center">
-          <h2 className="display-2">Summary</h2>
-        </div>
-        <div className="">
-          <h3 className="fw-light">
-            Your monthly budget is
-            <span className="text-primary">{` ${totBudget}€`}</span>
-          </h3>
-          <h3 className="fw-light">
-            You have spent
-            <span className="text-danger">{` ${totExpenses}€`}</span>
-          </h3>
-          <h3 className="fw-light">
-            This leaves you with
-            <span className="text-success">{` ${totSavings}€`}</span> until
-            <span>{' 31.03.22'}</span>
-          </h3>
-        </div>
-      </section>
-      <AddExpensesForm />
-    </div>
+    <section className="row my-5">
+      <div className="text-center">
+        <h2 className="display-2">Summary</h2>
+      </div>
+      <div className="">
+        <h3 className="fw-light">
+          Your monthly budget is
+          <span className="text-primary">{` ${totBudget}€`}</span>
```

Comment:
tot means death in german :)

Reply:
Yeah I thought the same thing. Maybe let's go with `totalBudget` etc.

Reply:
lol 👍